src/hooks/pages/configure/useNewDomainPage.ts

```diff
@@ -100,7 +101,10 @@ export function useNewDomainPage(): UseNewDomainPageReturn {
     setValue,
     formState: { errors },
   } = useForm({
-    defaultValues,
+    defaultValues: {
```

Comment:
calculate this in a useMemo. Cause we are creating a new object each time the component re renders